### ISSUE TITLE
Describe JS API in README.

### ISSUE THREAD

**Issue by rwjblue** [rwjblue/ember-template-lint#13]
``` js
var TemplateLinter = require('ember-template-lint');

var linter = new TemplateLinter();
var template = fs.readFileSync('some/path/to/template.hbs', { encoding: 'utf8' });
var results = linter.verify(template);
```

`results` above will be an array of "result" objects, which have the following properties:
- `rule` - The name of the rule that triggered this warning/error.
- `message` - The message that should be output.
- `line` - The line on which the error occurred.
- `column` - The column on which the error occurred.
- `moduleId` - The module path for the file containing the error.
- `source` - The source that caused the error.
- `fix` - An object describing how to fix the error.

**Comment by ksin** [comment]
@rwjblue would a "Usage" section mostly repeating what you described above be enough or are you looking for something more? Happy to make a quick PR.

**Comment by rwjblue** [comment]
Ya, I think that is a great start!

**Comment by ksin** [comment]
Is there a recommended file/directory to use the API in? (I'm actually trying to figure out how to use the addon right now...)

**Comment by rwjblue** [comment]
If you are using from an ember-cli app, the easiest and recommended way of using is `ember install ember-cli-template-lint` which should be all you need to start linting your apps templates. `ember-cli-template-lint` consumes `ember-template-lint` (this repo) and integrates it nicely with ember-cli.

If you are trying to use outside of the concept of ember-cli, this package is for you 😄, and the instructions in the issue description should mostly do the trick.

**Comment by ksin** [comment]
OH! Somehow I missed the ember-cli addon. But since I'm already here, I'll make a PR for the Usage section later today. :) Thanks for the help!
